utils/db.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Database
+  DB_PATH = './db/'
+  DB_FILE = 'db.yaml'
+
+  def data(action)
```

Comment:
I don't really like using `send` and raising an exception:
```
unless private_methods.include? :"#{action}_data"
  raise ArgumentError, 'Invalid action'
end
```
Let's remove `data` method and make other methods public (will be used like `library.save_data`)